src/SSD1306Spi.h

```diff
@@ -57,7 +58,8 @@ class SSD1306Spi : public OLEDDisplay {
 
     bool connect(){
       pinMode(_dc, OUTPUT);
-      pinMode(_cs, OUTPUT);
+      if (_cs != (uint8_t) -1)
+        pinMode(_cs, OUTPUT);
```

Comment:
```suggestion
      if (_cs != (uint8_t) -1) {
        pinMode(_cs, OUTPUT);
      }  
```

Reply:
I'm sorry. I just now realized that I had only adressed half of your concerns m(
Thanks for fixing that up, and it would have been 100% ok to just return this back to me for fixing :-)

Reply:
No worries 😄 I would have felt bad to nag you with such nit-pick. We do appreciate your contributions, thanks.